charts/mlp/values.yaml

```diff
@@ -1,4 +1,5 @@
-global: {}
+global:
+  protocol: "http"
```

Comment:
is it for authz only? or is it going to be used for the other components?

Reply:
This would apply to all, although this is a default value for a quick install.